docs/usage.rst

```diff
@@ -8,9 +8,57 @@ To use django-fperms-iscore in a project, add it to your `INSTALLED_APPS`:
 
     INSTALLED_APPS = (
         ...
-        'fperms_iscore.apps.FPermsIscoreConfig',
+        'fperms_iscore.apps.FPermsISCoreConfig',
         ...
     )
 
-    # set the extended model for FPerms
-    PERM_MODEL = 'fperms_iscore.IsCorePerm'
+
+Cores
+-----
+
+You can use `fperms_iscore.permissions.FPermPermission` with your cores to to set dynamic permissions system::
+
+
+.. code-block:: python
+
+    from is_core.main import UIRESTModelISCore
+    from is_core.auth.perm PermissionsSet
+    from fperms_iscore.permissions.FPermPermission
+
+    class UserISCore(UIRESTModelISCore):
+
+        model = User
+        permissions = PermissionsSet(
+            'create': FPermPermission('user__create', 'Create user'),
+            'read': FPermPermission('user__read', 'Read user'),
+            'update': FPermPermission('user__update', 'Update user'),
+            'delete': FPermPermission('user__delete', 'Delete user'),
+            'change_password': FPermPermission('user__change_password', register=False),
+        )
+
+
+As you can see class `fperms_iscore.permissions.FPermPermission` has two initial parameters. The first one is code name which is used for permission identification in the database and second one is a optional verbose name for better user experience. The last permission `change_password` has set parameter `register=False`, it means that permission will not be created but it will be used already defined permission with same code.
```

Comment:
- ... second one is **an** optional ...
- ... but it will **use** already defined ...